CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## [1.0.2](https://github.com/dhis2/multi-calendar-dates/compare/v1.0.1...v1.0.2) (2023-03-09)
```

Comment:
Now that there's an `alpha` h1 between this headline and the original `# 1.0.0` headline, should be remain an h2 or should this be an h1 as well?